pages/models.py

```diff
@@ -988,6 +988,19 @@ class InitiativeEndorsementPage(Page):
         ImageChooserPanel('social_image')
     ]
 
+    def _get_initiative_title(self):
+        """Build title for initiative and support legacy pages"""
+        if self.initiative_title and self.initiative_name:
+            initiative_title = (
+                str("Yes" if self.how_to_vote else "No") + ' on ' + self.initiative_title
+                + ': ' + self.initiative_name
+            )
+        else:
+            initiative_title = self.title
+
+        return initiative_title
+    get_initiative_title = property(_get_initiative_title)
```

Comment:
We don't really have a standard here but I usually use `get____` for methods and just a plain non-prefixed name for properties/fields. Like make a property called `display_title` or something like that?

This is also confusing because we have a field on the model called `initiative_title` but it's different from `get_initiative_title`. Can we make it more distinct name so they don't get mixed up?

Reply:
👍